book/getting-started/linux-setup.md

```diff
@@ -90,47 +51,26 @@ sudo update-alternatives --install /usr/bin/gcc gcc \
   /usr/bin/gcc-5 60 --slave /usr/bin/g++ g++ /usr/bin/g++-5
 ```
 
-### Install prce2
-
-#### Xenial Ubuntu:
-
-```bash
-sudo apt-get install -y libpcre2-dev
-```
-
-#### Trusty Ubuntu:
-
-*Note:* some older versions of Ubuntu do not supply a prce2
-package. If you get an error that no package exists (`E: Package
-'libpcre2-dev' has no installation candidate`) then you will need to
-install from source like this:
-
-```bash
-cd ~/
-wget ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre2-10.21.tar.bz2
-tar xjvf pcre2-10.21.tar.bz2
-cd pcre2-10.21
-./configure --prefix=/usr
-make
-sudo make install
-```
-
 ### Installing ponyc
 
-Now you need to install the Wallaroo Labs fork of the Pony compiler `ponyc`. Run:
+Now you need to install Pony compiler `ponyc`. Run:
 
 ```bash
-cd ~/
-wget https://github.com/WallarooLabs/ponyc/archive/wallaroolabs-19.2.14.tar.gz
-tar xzfv wallaroolabs-19.2.14.tar.gz
-cd ponyc-wallaroolabs-19.2.14
-sudo make config=release install
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "8756 C4F7 65C9 AC3C B6B8  5D62 379C E192 D401 AB61"
+echo "deb https://dl.bintray.com/pony-language/ponyc-debian pony-language main" | sudo tee -a /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get -V install ponyc
 ```
 
 You can check that the installation was successful by running:
```

Comment:
I think we could ditch this section?

Reply:
I'm not opposed to ditching it.  The only question is whether it helps to give them feedback the installation succeeded.  I think that probably compiling Wallaroo is a better form of that feedback (removing an extraneous step).